_includes/footer.html

```diff
@@ -48,7 +48,7 @@
         <div class="row">
             <p class="description small text-center">
                 Istio Version {{site.data.istio.version}}, Copyright &copy; 2017 Istio Authors<br>
-                This site was built on {{site.time | date_to_string | split: ' ' | join: '-'}}
+                Archived on {{site.data.istio.archive_date}}
```

Comment:
maybe this can be "Archive as of " vs "on" as it wasn't archived on / there are all kind of cosmetic changes on top / since ? or is that nit picking ?

the built date is still somewhat interesting but maybe that one shows at the toplevel of archive ?